src/gooddata.js

```diff
@@ -27,5 +28,5 @@ import * as config from './config';
  * @module sdk
  * @class sdk
  */
-export default { config, xhr, user, md, execution, project };
+export default { config, xhr, user, md, execution, project, catalogue };
```

Comment:
I'm not sure how we could use that without exposing? I would not go the way that only users of npm version can use part of the SDK functionality...